nl/firefox-ios.xliff

```diff
@@ -1044,6 +1042,10 @@
         <target>Downloads worden nog niet ondersteund in Firefox.</target>
         <note>The message displayed to a user when they try and perform the download of an asset that Firefox cannot currently handle.</note>
       </trans-unit>
+      <trans-unit id="Enable Push">
```

Comment:
@st3fan Ask Jean / Vijay if these strings actually need to be in here. My guess is not.